shell/platform/android/io/flutter/embedding/engine/mutatorsstack/FlutterMutatorView.java

```diff
@@ -88,8 +103,27 @@ public void dispatchDraw(Canvas canvas) {
     // all the clipping paths
     finalMatrix.postTranslate(-left, -top);
 
-    canvas.concat(finalMatrix);
-    super.dispatchDraw(canvas);
-    canvas.restore();
+    return finalMatrix;
   }
+
+  @Override
+  // Intercept the events here and do not propagate them to the child platform views.
```

Comment:
minor nit: for consistency, what about moving the `@Override` after the javadoc, and using `/** */`?

Reply:
sounds good